src/net.cpp

```diff
@@ -681,16 +690,16 @@ bool CNode::ReceiveMsgBytes(Span<const uint8_t> msg_bytes, bool& complete)
     nRecvBytes += msg_bytes.size();
     while (msg_bytes.size() > 0) {
         // absorb network data
-        int handled = m_deserializer->Read(msg_bytes);
-        if (handled < 0) {
+        bool handled = m_transport->ReceivedBytes(msg_bytes);
+        if (!handled) {
```

Comment:
nit (`handled` is not used afterwards):
```suggestion
        if (!m_transport->ReceivedBytes(msg_bytes)) {
```
(same in `p2p_transport_serialization.cpp`)

Reply:
I don't mind keeping `handled` around, to make it more clear that receiving 0 bytes is also "handled".

Reply:
nit: if `handled` is decided to be kept, it should be declared as `const`.